recipes/guiqwt/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "guiqwt" %}
+{% set version = "3.0.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PierreRaybaut/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: fcad0c1745c2685d6dcc3367ba6c5fa5dbbcd5991bf005e787c36e8e49b520a2
+  fn: {{ name }}-{{ version }}.tar.gz
+
+  patches:
+    # Needed to avoid some random exceptions which can make the application
+    # crash
+    - curve.patch
+
+build:
+  number: 0
+  skip: true  # [win and vc!=14]
```

Comment:
I think you don't need it anymore, do you?

Reply:
Removed!